crypto/bn256/cloudflare/gfp6.go

```diff
@@ -168,7 +168,7 @@ func (e *gfP6) Square(a *gfP6) *gfP6 {
 	return e
 }
 
-func (e *gfP6) Invert(a *gfP6) *gfP6 {
+func (e *gfP6) InvertConstantTime(a *gfP6) *gfP6 {
```

Comment:
Afaict from upstream, where they want the default to be constant time, I suppose you should rename `InvertConstantTime` back to `Invert`, and have `InvertVariableTime` be the non-default options.